api/app/weather_models/fetch/predictions.py

```diff
@@ -124,10 +112,9 @@ async def fetch_latest_daily_model_run_predictions_by_station_code_and_date_rang
         return results
 
 
-async def fetch_latest_model_run_predictions_by_station_code_and_date_range(session: Session,
-                                                                            station_codes: List[int],
-                                                                            start_time: datetime.datetime,
-                                                                            end_time: datetime.datetime) -> List[WeatherIndeterminate]:
+async def fetch_latest_model_run_predictions_by_station_code_and_date_range(
```

Comment:
What do you think about logging execution time for this code? Or maybe at the route level?

Reply:
Done in: https://github.com/bcgov/wps/pull/4118/commits/8b6dd8c7c493bfa8a2fccf7a7113eab96fa44d5b

Reply:
Thank you!